operations/utility_storage/Alternate_Storage_Pools.md

```diff
@@ -0,0 +1,187 @@
+# Alternate Storage Pools
+
+## Description
+
+Creating, maintaining, and removing Ceph storage pools.
+
+## Use cases
+
+* A landing space for the CSM tarball used for upgrades.
+* Temporary space needed for maintenance or pre/post upgrade activities.
+
+## Best practices
+
+* Apply a proper quota to any pools created.
+  * This will use storage from the default CRUSH rule which is utilizing every OSD.
+  * Improper use and policing of space usage can have a negative impact on the cluster and space available to running services that have determined the space provided by the design and sizing exercises.
+* Cleanup after the criteria for the pool creation has been met.
+  * This can be as simple as removing volumes but leaving the pool for future use.
+
+## Procedures
+
+This example shows the creation and mounting of an `rbd` device on `ncn-m001`.
+
+**NOTE:** The commands to create and delete pools or `rbd` devices must be run from a master node or one of the first three storage nodes (`ncn-s001`, `ncn-s002`, or `ncn-s003`).
+
+### Create storage pool
+
+The below example will create a storage pool name `csm-release`. The pool name can be changed to better reflect any use cases outside of support for upgrades.
+
+```bash
+ceph osd pool create csm-release 3 3
+ceph osd pool application enable csm-release rbd
+ceph osd pool set-quota csm-release max_bytes 500G
+ceph osd pool get-quota csm-release
+```
+
+Output:
+
+```text
+ncn-s001:~ # ceph osd pool create csm-release 3 3
+pool 'csm-release' created
+
+ncn-s001:~ # ceph osd pool application enable csm-release rbd
+enabled application 'rbd' on pool 'csm-release'
+
+ncn-s001:~ # ceph osd pool set-quota csm-release max_bytes 500G
+set-quota max_bytes = 536870912000 for pool csm-release
+
+ncn-s001:~ # ceph osd pool get-quota csm-release
+quotas for pool 'csm-release':
+  max objects: N/A
+  max bytes  : 500 GiB  (current num bytes: 0 bytes)
+```
+
+**NOTES:**
+
+* The above example sets the quota to 500 GiB.
+  * If this pool is fully utilized it will be using 1.5 TiB of raw space.
+  * This space counts against the total space provided by the cluster; Use cautiously.
+  * If this pool or any pool reaches 95-100% utilization, then all volumes for the fully utilized pool will go into read-only mode.
+
+### Create and map an `rbd` device
+
+**IMPORTANT:**
+
+* Creating an `rbd` device requires proper access and must be run from a master node or one of the first three storage nodes (`ncn-s001`, `ncn-s002`, or `ncn-s003`).
+* Mounting a device will occur on the node where the storage needs to be present.
```

Comment:
Can it be mounted on multiple nodes simultaneously?

Reply:
So technically it can mounted RW on one node and RO on other nodes simultaneously.  The caveat there is it requires a lot more documentation as a write to the rbd device will cause the RO nodes to enter a weird state that will require re-mounting the FS.  This is due to a lack of clustered filesystem software.